mk/llvm.mk

```diff
@@ -83,7 +83,7 @@ endif
 LLVM_LINKAGE_PATH_$(1):=$$(abspath $$(RT_OUTPUT_DIR_$(1))/llvmdeps.rs)
 $$(LLVM_LINKAGE_PATH_$(1)): $(S)src/etc/mklldeps.py $$(LLVM_CONFIG_$(1))
 	$(Q)$(CFG_PYTHON) "$$<" "$$@" "$$(LLVM_COMPONENTS)" "$$(CFG_ENABLE_LLVM_STATIC_STDCPP)" \
-		$$(LLVM_CONFIG_$(1))
+	        "$$(CFG_ENABLE_LLVM_LIBCPP)" $$(LLVM_CONFIG_$(1))
```

Comment:
Actually it's worse than that: `configure` doesn't even `putvar` anything related to `libcpp`. Nice catch, thanks.